ArduPlane/Log.cpp

```diff
@@ -142,11 +143,12 @@ void Plane::Log_Write_OFG_Guided()
         time_us                : AP_HAL::micros64(),
         target_airspeed_cm     : (float)guided_state.target_airspeed_cm*(float)0.01,
         target_airspeed_accel  : guided_state.target_airspeed_accel,
-        target_alt             : guided_state.target_alt,
+        target_alt             : guided_state.target_location.alt * 0.01,
         target_alt_accel       : guided_state.target_alt_accel,
-        target_alt_frame       : guided_state.target_alt_frame,
+        target_mav_frame       : guided_state.target_mav_frame,
         target_heading         : guided_state.target_heading,
-        target_heading_limit   : guided_state.target_heading_accel_limit
+        target_heading_limit   : guided_state.target_heading_accel_limit,
+        target_alt_frame       : static_cast<uint8_t>(guided_state.target_location.get_alt_frame()),
```

Comment:
rearrangment of field order is a little unfortunate here

Reply:
It's not a reordering in terms of the values that end up in the log. The 6th field used to be called target_alt_frame but it contained the MAVLINK frame, so now the 6th field still contains the mavlink frame, the same value at the same position in the log and still called AltF - but it's source is a different value target_mav_frame.